### ISSUE TITLE
No configuration for the debug configuration type launch snapshotInitialize is available

### ISSUE THREAD

**Issue by CarlosAndreuLlaneras** [microsoft/AL#6290]
- Issue description: I am facing this error when I try to download symbols or publish an extension. I have realized that the problem only occurs if I am using a workspace. 

- AL Extension Version: v6.1.362735

- Server Version: ES Business Central 16.5 (Plataforma 16.0.15884.15941 + Aplicación 16.5.15953.0)

- Steps to reproduce the issue:
I've created two extensions using "AL:Go..." option in VSCode. If I open separately everyone in a diferente VSCode instance, I can download symbols, but If I create a workspace including both folders and I try to download symbols in any of them the OUTPUT shows "Error: No server has been chosen." and the AL Language launches the error: "No configuration for the debug configuration type launch snapshotInitialize is available".

I've attached an example (AL workspace with two AL extensions)

![image](https://user-images.githubusercontent.com/53298196/98542779-bc519e80-2291-11eb-9cc9-ebd9a277d12b.png)

[Workspace Sample.zip](https://github.com/microsoft/AL/files/5510592/Workspace.Sample.zip)

Regards,
Carlos

**Comment by plagijat** [comment]
Same here.

I also have this issue when working with workspace.

Zoran M.

**Comment by CarlosAndreuLlaneras** [comment]
Oops I forgot to mention a little detail: I am getting this issue since I updated VSCode to the last version (v1.51) and rebooted the laptop.   ;-)

**Comment by kalberes** [comment]
Yes, it looks like something got broken in my opinion in vscode's configuration code base.
Naming and saving the workspace and (unfortunately) restarting vscode should fix the issue.

**Comment by CarlosAndreuLlaneras** [comment]
This workaround is not working for my.
I've renamed and recreated the workspace (restarting VSC every time) but I always get the same error. 

Regards

**Comment by ajkauffmann** [comment]
Had the same problem, and I was able to solve it by adding a launch configuration to the workspace file.

![image](https://user-images.githubusercontent.com/19155831/98690655-7407ae80-236d-11eb-8adf-b61923f7c940.png)

**Comment by CarlosAndreuLlaneras** [comment]
Yes, adding one launch configuration at workspace file, works for me too. 
But, if you add more than one launch configuration, it fails again. As I use several launch configurations, I prefer to open each folder in a different VSCode instance.

**Comment by Easystep2** [comment]
Nothing seems to help can't even Debug an Customer Sandbox at the moment, keep getting this message
**"No configuration for the debug configuration type launch attach  is available"** (assume its related)

Is there an solution for this soon? Even tried creating an new project from scratch and have the same issue, can we downgrade or have an workaround for this so I can help my customers!!

**Comment by kalberes** [comment]
@Easystep2 
Can we have a concise repro step. I have 1.51.1 and also 1.52.insider (green vscode)
I have a workspace with 2 projects each having its own launch.json in the .vscode folder.
After creating the workspace I have restarted vscode on 1.51. I have tried without restarting on 1.51.1 and had no issues.
I have many configurations in both my launch.json files.
I have no issues downloading symbols, debugging, publishing. Or any other command that requires that we connect to  a server using the configuration settings,
